packages/voiceflow-types/src/node/index.ts

```diff
@@ -11,13 +12,22 @@ import * as Speak from './speak';
 export * as Buttons from './buttons';
 export * as Capture from './capture';
 export * as CaptureV2 from './captureV2';
+export * as CardV2 from './cardV2';
 export * as Carousel from './carousel';
 export * as Interaction from './interaction';
 export * as Prompt from './prompt';
 export * as Speak from './speak';
 
-export type AnyExtendedStep = Speak.Step | Prompt.Step | Capture.Step | CaptureV2.Step | Interaction.Step | Buttons.Step | Carousel.Step;
-export type AnyExtendedNode = Speak.Node | Capture.Node | CaptureV2.Node | Interaction.Node | Carousel.Node;
+export type AnyExtendedStep =
+  | Speak.Step
+  | Prompt.Step
+  | Capture.Step
+  | CaptureV2.Step
+  | Interaction.Step
+  | Buttons.Step
+  | Carousel.Step
+  | CardV2.Step;
+export type AnyExtendedNode = Speak.Node | Capture.Node | CaptureV2.Node | Interaction.Node | Carousel.Node | CardV2.Node;
```

Comment:
could remove node from here too
```suggestion
export type AnyExtendedNode = Speak.Node | Capture.Node | CaptureV2.Node | Interaction.Node | Carousel.Node;
```